tests/installation/addon_products_yast2.pm

```diff
@@ -78,6 +81,7 @@ sub run {
                 send_key 'alt-o';
             }
             if (check_screen 'addon-installation-pop-up', 100) {                    # e.g. RT reboot to activate new kernel
```

Comment:
I know that this code was here before, but it's bad that we will wait for 100 seconds when pop up is not there. So maybe we can improve here and do assert_screen on multiple tags and process them accordingly using match_has_tag api call. If we don't have time for that now, maybe we can create progress ticket for that.

Reply:
yes, it would be nice, but I don't know how to do that so I filled https://progress.opensuse.org/issues/25754